spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -322,9 +301,7 @@
         variant.update on_hand: 5
         visit shop_path
 
-        accept_alert 'Insufficient stock available, only 5 remaining' do
-          fill_in "variants[#{variant.id}]", with: '10'
-        end
+        click_add_to_cart variant, 5
```

Comment:
Why did the behaviour change? Is there not an alert anymore? why?

Reply:
Oh, well spotted! That's a mistake. In the new mobile UX it won't be possible to click on the button once the stock limit is reached. But this spec change should happen then, not now. I will change it.